src/utils/time.js

```diff
@@ -1,72 +1,243 @@
-import moment from 'moment';
+import moment from 'moment-timezone';
+import {head, last} from './func';
 
-const getCurrentTime = () => moment().format("HH:mm");
+// loads moment-timezone's timezone data, which comes from the
+// IANA Time Zone Database at https://www.iana.org/time-zones
+moment.tz.load({version: 'latest', zones: [], links: []});
 
+/**
```

Comment:
Added a bunch of docstrings in here since it's a time thing and this is the ❤️ of all of that functionality, really. This function takes user props that are passed to it and returns an object with all the data needed to construct a valid time, including timezone information.